src/getHumanAge.test.js

```diff
@@ -1,10 +1,60 @@
 'use strict';
 
-describe('getHumanAge', () => {
+describe(`getHumanAge`, () => {
```

Comment:
Use single quotes instead of backticks for strings without template literals or variables. Replace `getHumanAge` with 'getHumanAge'. 📝